File: src/main/java/net/finance/config/NoRedirectStrategy.java
package net.finance.config;

import java.io.IOException;

import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

import org.springframework.security.web.RedirectStrategy;

public class NoRedirectStrategy implements RedirectStrategy {

	@Override
	public void sendRedirect(final HttpServletRequest arg0, final HttpServletResponse arg1, final String arg2) throws IOException {
		// TODO No redirect is required
	}

}

File: src/main/kotlin/com/finances/config/NoRedirectStrategy.kt
package com.finances.config

import org.springframework.security.web.RedirectStrategy
import java.io.IOException
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

class NoRedirectStrategy : RedirectStrategy {
    @Throws(IOException::class)
    override fun sendRedirect(arg0: HttpServletRequest, arg1: HttpServletResponse, arg2: String) {
        // TODO No redirect is required
    }
}